.gitignore

```diff
@@ -0,0 +1 @@
+*test.md
```

Comment:
i added linters the way it was described in the repo, what could be the possible reason why it is not reflecting, is there a way you can point me in the direction of what i am not adding or doing wrongly

Reply:
@sumairq this is me tagging you to the last question i asked

Reply:
@sumairq i also don't have a node_modules folder or its containing file that is the basic reason why i didnt add it in the gitignore.

Reply:
The tests are running now, therefore it's good to go now. 👍

Reply:
In software development, you will be using node and npm for installing dependencies, and you will always be including the node_modules directory in the `.gitignore` file. 
Even though you might not have a node_modules directory it can be accidentally introduced while installing linters using npm, therefore it is a requirement to include it in your readme file.

Reply:
Thank you @sumairq i have passed all the linters test, can i connect with you on the slack channel tho

Reply:
can i merge the pull request or i need to submit a new review request @sumairq

Reply:
You need to submit a new pull request for the re-review . Once the PR is approved only then you can merge the pull request.

Reply:
Sure, feel free to connect on slack. 👍

Reply:
i cant seem to find where i am to resubmit, is it from github here or my dashboard and you asked me to make changes to the title of my pull request, how will i do that if i am submitting the same pull request ? @sumairq 

i cant also find you on slack